src/lib/models/licence.js

```diff
@@ -36,6 +40,22 @@ class Licence {
   get licenceNumber () {
     return this._licenceNumber;
   }
+
+  set accountNumber (accountNumber) {
```

Comment:
Do we need this when we already have the InvoiceLicence and InvoiceAccount models